src/prost/build.rs

```diff
@@ -87,7 +103,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .expect("Failed to compile grpc!");
 
     // Implement `serde::Serialize` on those structs.
-    let descriptor_set = std::fs::read(file_descriptor_set_path)?;
+    let descriptor_set = fs_err::read(file_descriptor_set_path)?;
     pbjson_build::Builder::new()
         .register_descriptors(&descriptor_set)?
         .out_dir(out_dir.as_path())
```

Comment:
Can we directly let it output to the `src` directory? 🤔 Will it handle `compare_and_copy` for us?

Reply:
I don't get it. Isn't this the old behavior?

Reply:
Oops. My fault.